### PR TITLE
xero.gg | Added settings.

### PR DESCRIPTION
**Pull request by Dekirai**
Added 2 settings to the xero.gg presence.
• Show chatting opponent
• Show profile names

Showcase of it working: [Video](https://kevin.s-ul.eu/BPYU9Qic.mp4)

### CODE REVIEW

File: websites/X/Xero.gg/presence.ts [hunk 1]
```diff
@@ -14,20 +14,27 @@ presence.on("UpdateData", async () => {
     largeImageKey: "xero"
   };
 
+  const showChat = await presence.getSetting("showchat");
+  const showProfile = await presence.getSetting("showprofile");
```
Comment (ririxichan) [inline websites/X/Xero.gg/presence.ts:18]: merge consts

**Review by ririxichan — CHANGES_REQUESTED** [review]

**Comment by Dekirai** [comment]
Huh, whats wrong now?

**Comment by ririxichan** [comment]
Nothing, codacy just reporting thing taht you fixed

**Comment by Dekirai** [comment]
Oh, okay, lol

**Comment by Dekirai** [comment]
Can I fix this codacy error somehow or..?
Since it tells me to merge the consts which are already merged.. dunno.

**Comment by ririxichan** [comment]
It just bugged, dw.

**Comment by Dekirai** [comment]
Alright, thank you.

**Review by Bas950 — CHANGES_REQUESTED** [review]
V1.1.0 not 1.1.1

**Comment by Dekirai** [comment]
Aaaaaa, I don't understand codacy. :(

File: websites/X/Xero.gg/presence.ts [hunk 2]
```diff
@@ -14,20 +14,27 @@ presence.on("UpdateData", async () => {
     largeImageKey: "xero"
   };
 
+  const showChat = await presence.getSetting("showchat"),
```
Comment (ririxichan) [inline websites/X/Xero.gg/presence.ts:17]: bruh yeah, combine it with presenceData const (i didn't saw it because it was hidden)

**Review by ririxichan — CHANGES_REQUESTED** [review]

File: websites/X/Xero.gg/presence.ts (hunk 2 above)
Comment (Dekirai) [reply websites/X/Xero.gg/presence.ts:17]: OOOOOH, makes sense, I didn't notice it either...

**Review by Bas950 — APPROVED** [review]

**Review by ririxichan — APPROVED** [review]

**Pull request merged**